src/telliot_feeds/queries/query_catalog.py

```diff
@@ -18,32 +19,6 @@
 # Query Catalog Entries
 # --------------------------------------------------------------------------------------
 
-query_catalog.add_entry(
-    tag="eth-usd-legacy",
-    title="Legacy ETH/USD spot price",
-    q=LegacyRequest(legacy_id=1),
-)
-
-query_catalog.add_entry(
-    tag="btc-usd-legacy",
-    title="Legacy BTC/USD spot price",
-    q=LegacyRequest(legacy_id=2),
-)
-
-query_catalog.add_entry(
-    tag="ampl-legacy",
-    title="Legacy AMPL/USD custom price",
-    q=LegacyRequest(legacy_id=10),
-)
-
-query_catalog.add_entry(tag="uspce-legacy", title="Legacy USPCE value", q=LegacyRequest(legacy_id=41))
-
-query_catalog.add_entry(
-    tag="trb-usd-legacy",
-    title="Legacy TRB/USD spot price",
-    q=LegacyRequest(legacy_id=50),
-)
-
```

Comment:
I guess if these are going to be removed might as well remove ```LEGACY_DATAFEEDS``` from the init file, right?

Reply:
Removed, thanks for spotting.